paparazzi/paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -89,7 +93,8 @@ class Paparazzi @JvmOverloads constructor(
   private val maxPercentDifference: Double = 0.1,
   private val snapshotHandler: SnapshotHandler = determineHandler(maxPercentDifference),
   private val renderExtensions: Set<RenderExtension> = setOf(),
-  private val supportsRtl: Boolean = false
+  private val supportsRtl: Boolean = false,
+  private val viewOnly: Boolean = false
```

Comment:
`viewOnly` is reminding me of file attributes such as `read only`. 

Considering that our usages of the term `View` is going to only go down as we slowly write everything as composables, can I propose using something else instead? maybe `hideDeviceFrame`?

Reply:
Good call. Let’s workshop a new name.

Two consequences:
 - no frame
 - no device dimensions

Any other suggestions?

Reply:
Another suggestion is to match what Android Studio uses for emulators -- `enableDeviceFrame`